test/test_response.py

```diff
@@ -2,8 +2,12 @@
 
 from io import BytesIO, BufferedReader
 
+try:
+    import http.client as httplib
+except:
```

Comment:
Let's do this as `except ImportError` as well. Naked `except` statements are generally pretty bad because they catch all sorts of things you aren't expecting (for example, `KeyboardInterrupt`!).